.eslintrc.json

```diff
@@ -1,3 +1,24 @@
 {
-  "extends": "amex"
+  "extends": "amex",
```

Comment:
```suggestion
  "extends": "amex/prettier",
```

Reply:
There should be a separate effort for this.